public/static/app/perfdiscovery/PerformanceDiscoveryCtrl.js

```diff
@@ -116,10 +116,22 @@ mciModule.controller('PerformanceDiscoveryCtrl', function(
         return dataUtil.getBFTicketsForRows(vm.gridOptions.data)
       })
       // Apply BF tickets to rows data
-      .then(function(ticketsForTasks) {
-        _.each(ticketsForTasks, function(v, k) {
-          _.each(_.where(vm.gridOptions.data, {taskId: k}), function(task) {
-            task.buildFailures = v
+      .then(function(bfGroups) {
+
+        // Match grouped points (by task, bv, test) w/ rows
+        _.each(bfGroups, function(bfs) {
```

Comment:
We probably want to dedupe BFs here since it looks like this data structure can contain the same ticket multiple times

Reply:
Each BF ticket in each group is unique across all BF tickets in the set. If you look at the screenshot, you'll look "duplicates", because the same ticket might be assigned to different tests within the same task. BFs which have `test` property, will be assigned to single row, but most BFs doen't have `test` field.

Reply:
I'm talking about for the same test. If you run this locally and put sys_perf_e573d7f2f908f3fbe96716851cd1b1e3d65fe7c9 as the version for the baseline and comparison, you can see this (BF-9580 is also open but has a strikethrough):
![image](https://user-images.githubusercontent.com/29384396/41783925-22c5231a-760c-11e8-8e3a-5f73b3d7d15f.png)